lib/pdf_fill/forms/form_helper.rb

```diff
@@ -43,6 +45,7 @@ def split_postal_code(address)
         return if address.blank?
 
         postal_code = address['postalCode']
+        postal_code = address['zip_code'] if postal_code.blank?
```

Comment:
Does this work?

`postal_code = address['postalCode'] || address['zip_code']`

if so you could use that pattern in other areas like this :)

Reply:
I've updated this as suggested - thanks!